edx_credentials_themes/templates/mitpe/credentials/programs/base.html

```diff
@@ -26,17 +27,17 @@
       <div class="wrapper-accomplishment-statement">
         <div class="accomplishment-statement">
           <p class="accomplishment-statement-lead">
-            <span class="accomplishment-statement-detail copy">{% trans "This is to certify that" %}</span>
+            <span class="accomplishment-statement-detail copy">{% trans "This is to certify that" as tmsg %}{{ tmsg | htmlescape }}</span>
             <strong class="accomplishment-recipient">{{ user_data.name }}</strong>
             <img class="accomplishment-logo" src="{% static 'mitpe/images/mit-seal.png' %}"
                  alt="Massachusetts Institute of Technology">
-            <span class="accomplishment-summary copy">{% trans "has successfully completed the" %}</span>
+            <span class="accomplishment-summary copy">{% trans "has successfully completed the" as tmsg %}{{ tmsg | htmlescape }}</span>
```

Comment:
Words like the/a in Spanish are difficult to translate without knowing what comes after them (due to gender). Though we might be able to add a comment to provide that context for translators, I suspect we might have issues if/when this exact string appears in another template. It might be good to also provide a `context` argument to the `trans` tag so that we can specify that this translation is to be used only when the string appears directly before 'professional certificate program'. See https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#contextual-markers

Reply:
@amangano-edx this is an mitpe credential template, which the ticket lists as UNGOALS. I can add translator context if we now think that should be a part of this work (I added escaping since it the credential type doesn't really impact the possibility of vulnerability)

Reply:
I think the escaping of mitpe strings definitely should be part of this work, but you raise a good point about whether or not we should fix strings in mitpe templates to make them easier to translate. I think if you want to punt on that it's probably fine. Maybe just create a JIRA ticket noting the known issues and potential fix (using the context argument) so that they don't get lost?

Reply:
👍